
Well - this is just to get my ARM build going - we should fill them soon ;-)

  - Rene Rebe <rene@exactcode.de>

--- /dev/null	1970-01-01 01:00:00.000000000 +0100
+++ dietlibc-0.27/libm/sin.c	2005-02-02 14:42:19.191367320 +0100
@@ -0,0 +1,8 @@
+#include <math.h>
+#include <stdlib.h>
+
+double  sin ( double x )
+{
+    __write2("No sin yet.");
+    exit (1);
+}
--- /dev/null	1970-01-01 01:00:00.000000000 +0100
+++ dietlibc-0.27/libm/cos.c	2005-02-02 14:41:58.627493504 +0100
@@ -0,0 +1,8 @@
+#include <math.h>
+#include <stdlib.h>
+
+double  cos ( double x )
+{
+    __write2("No sin yet.");
+    exit (1);
+}
--- /dev/null	1970-01-01 01:00:00.000000000 +0100
+++ dietlibc-0.27/libm/tan.c	2005-02-02 14:43:21.313923256 +0100
@@ -0,0 +1,8 @@
+#include <math.h>
+#include <stdlib.h>
+
+double  tan ( double x )
+{
+    __write2("No tan yet.");
+    exit (1);
+}
